Games/types/Mafia/roles/cards/Eavesdrop.js

```diff
@@ -0,0 +1,51 @@
+const Card = require("../../Card");
+const { PRIORITY_DAY_DEFAULT } = require("../../const/Priority");
+const Message = require("../../../../core/Message");
+module.exports = class Eavesdrop extends Card {
+
+    constructor(role) {
+        super(role);
+
+        this.meetings = {
+            "Eavesdrop On": {
+                states: ["Day"],
+                flags: ["voting"],
```

Comment:
you can add "mustAct" to sweep the "novote" issue under

Reply:
created an action to clear the stalk target each night to resolve this issue